packages/core/src/Page/index.ts

```diff
@@ -170,6 +170,33 @@ export class Page {
     };
   }
 
+  /**
+   * Filters out icon asset files that are not used in a page.
+   * @param preVueSsrHtml html content of the page before processing Vue components
```

Comment:
How about summarize the rationale you gave me in https://github.com/MarkBind/markbind/pull/2357#issuecomment-1752266692 here, so that future devs will know why we are scanning both html content?